pkg/iosrc/file.go

```diff
@@ -38,6 +38,18 @@ func (s *FileSource) WriteFile(_ context.Context, d []byte, uri URI) error {
 	return wrapfileError(uri, err)
 }
 
+func (s *FileSource) WriteFileIfNotExists(_ context.Context, b []byte, uri URI) error {
```

Comment:
Nit: All the other `WriteFile` and `WriteFileIfNotExists` implementations call the second parameter `d` instead of `b`.
```suggestion
func (s *FileSource) WriteFileIfNotExists(_ context.Context, d []byte, uri URI) error {
```

Reply:
This stuff's all gonna change so let's not spend time on nits right now.   (Plus I prefer changing them all to b in line with Go's io package.)